plugin/black.vim

```diff
@@ -19,16 +19,28 @@ if exists("g:load_black")
   finish
 endif
 
-if v:version < 700 || !has('python3')
-    func! __BLACK_MISSING()
-        echo "The black.vim plugin requires vim7.0+ with Python 3.6 support."
+" check if vim version is too old
+if v:version < 700
+    func! __BlackNotSupported()
+        echo "The black.vim plugin requires Vim 7.0+"
     endfunc
-    command! Black :call __BLACK_MISSING()
-    command! BlackUpgrade :call __BLACK_MISSING()
-    command! BlackVersion :call __BLACK_MISSING()
-    finish
 endif
 
+" check if Vim has Python 3 support
+if !has('python3')
+    func! __BlackNotSupported()
+        echo "The black.vim plugin requires Vim compiled with Python 3.8+ support."
+    endfunc
+endif
```

Comment:
This will hide the version requirements if python3 support is not available. I think it will be more useful to use:

    if v:version < 700
        ...
    elseif !has('python3')
        ...
    endif

Of course with this if you had very old vim and you get vim 7+ without python3 support you will find about the python3 support on the second try. The current error message avoid this issue by displaying both requirements, but fail to explain what is the actual issue.

We can do something like this:

Vim is too old:

    Vim {version} is too old.
    The black.vim plugin requires Vim 7.0+ with python3 support.

Python3 is missing:

    Python3 support is not available.
    The black.vim plugin requires Vim 7.0+ with python3 support.